dashboard/lib/projects_list.rb

```diff
@@ -60,7 +60,7 @@ def fetch_section_projects(section)
     #   which to search for the requested projects. Must not be specified
     #   when requesting all project types. Optional.
     # @return [Hash<Array<Hash>>] A hash of lists of published projects.
-    def fetch_published_projects(project_group, limit:, published_before:)
+    def fetch_published_projects(project_group, limit:, published_before:, prepend_featured:)
```

Comment:
worth testing this param since it's now going to be used on a user facing page?

Reply:
The name of this flag is really useful, makes it clear that they aren't all featured.

Reply:
I'm hoping this param only lives until the end of the week max, and there's test coverage to ensure the response is successful without the param.